[docs] explain initial value for bind scroll (#7065)
sveltejs/svelte: site/content/docs/02-template-syntax.md

<svelte:window bind:scrollY={y}/>
```

> > Note that only subsequent changes to the bound variable of `scrollX` and `scrollY` will be scrolled automatically. To scroll initially, use `scrollTo()` in `onMount()`.

Author: ```suggestion
> Note that only subsequent changes to the bound variable of `scrollX` and `scrollY` will cause scrolling automatically. The initial value will not cause scrolling as it brings accessibility issues. However if the scrolling behaviour is desired, call `scrollTo()` in `onMount()` instead.
```

> > Note that only subsequent changes to the bound variable of `scrollX` and `scrollY` will cause scrolling automatically. The initial value will not cause scrolling as it brings accessibility issues. However if the scrolling behaviour is desired, call `scrollTo()` in `onMount()` instead.

Reviewer: just a minor suggestion to reword. the content all looks good to me
```suggestion
> Note that the page will not be scrolled to the initial value to avoid accessibility issues. Only subsequent changes to the bound variable of `scrollX` and `scrollY` will cause scrolling. However, if the scrolling behaviour is desired, call `scrollTo()` in `onMount()`.
```
